### PR TITLE
Adding Llama2 fine tuning on Slurm

### PR DESCRIPTION
**Pull request by jrossthomson**
Looking to get the Llama 2 fine tuning published. Based on work by Injae Kwak.

Cleaned up the conda install. Detailed instructions.  HPC Toolkit page also to be published.

### CODE REVIEW

File: llama2-finetuning-slurm/README.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+
+![eula](images/image2.png "llama2 EULA")
+
+
+    Click "I HAVE READ AND ACCEPT THE LICENSE FOR LLAMA 2".
+
+2. Download the model with `gsutil`. In your home directory on the login node run the command.
+
+gsutil -m cp -r gs://vertex-model-garden-public-us-central1/llama2/llama2-7b-hf/ .
```
Comment (wkharold) [inline llama2-finetuning-slurm/README.md:138]: Alternative `gcloud` download command
```bash
gcloud storage cp --recursive gs://vertex-model-garden-public-us-central1/llama2/llama2-7b-hf/ .
```

File: llama2-finetuning-slurm/README.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+
+### Connect to the Slurm HPC cluster
+
+To run the Llama 2 fine tuning on your cluster, you must login to the Slurm _login _node. This is done by clicking on the `SSH` button using the Google Cloud Console.
+
+
+1. Connect to the Console at https://console.cloud.google.com/compute/instances
+2. Click on `SSH` next to the `login` node
+
```
Comment (wkharold) [inline llama2-finetuning-slurm/README.md:119]: Go direct to the login node via
```bash
gcloud compute ssh $(gcloud compute instances list --filter "name ~ login" --format "value(name)") --tunnel-through-iap --zone us-east1-b
```

**Review by wkharold — CHANGES_REQUESTED** [review]
I was not able to get any L4s, I do have the quota, but everything else seemed to work as expected. I like using the `gcloud` commands, it's up to you.

File: llama2-finetuning-slurm/README.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+1. Setup the  [Cloud HPC Toolkit](https://cloud.google.com/hpc-toolkit/docs/setup/configure-environment).
+2. Clone the Github repository.
+
+           git clone https://github.com/GoogleCloudPlatform/scientific-computing-examples.git
+
+
+
+3. Change to the AI Infrastructure directory
+
+    cd ai-infrastructure
```
[the remark this answers is not in the document]
Comment (jrossthomson) [reply llama2-finetuning-slurm/README.md]: Done

**Review by wkharold — APPROVED** [review]
LGTM

**Pull request merged**